zerver/lib/test_helpers.py

```diff
@@ -365,3 +365,5 @@ def send_json_payload(self, email, url, payload, stream_name=None, **post_params
 
         return msg
 
+    def get_recently_added_message(self):
+        return Message.objects.filter().order_by('-id')[0]
```

Comment:
Would probably name this `get_last_message`, and implement it using the `Message.objects.latest('id')` version since that's a bit cleaner.

Also commit message should probably hint that this is test-related, e.g. "test_helpers: Create get_last_message helper."